deploy/K8s-onprem-MIG/flower-hpa.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+    name: flower-hpa
+spec:
+    scaleTargetRef:
+      apiVersion: apps/v1beta1
+      kind: Deployment
+      name: flower
+    minReplicas: 1
+    maxReplicas: 7
+    metrics:
+    # use a "Pods" metric, which takes the average of the
+    # given metric across all pods controlled by the autoscaling target
+    - type: Pods
+      pods:
+        metricName: avg_time_queue_ms
+        targetAverageValue: 50
```

Comment:
Seems very model specific. Can all the fixed values become variables and then the README can describe what they mean.

Reply:
Explained in Readme that users can define their own custom metric for HPA.